app/templates/server/api/user(auth)/user.model.spec(mongooseModels).js

```diff
@@ -14,18 +14,12 @@ var genUser = function() {
 };
 
 describe('User Model', function() {
-  before(function() {
-    // Clear users before testing
-    return User.removeAsync();
-  });
+  // Clear users before testing
+  before(User.removeAsync);
```

Comment:
@deltreey my apologies, I should have mentioned that the `User` methods will loose context to their parent object, so they'll need to be bound like: `User.removeAsync.bind(User)`